working dir/widgets/terminal/Terminal.kt

```diff
@@ -35,11 +36,11 @@ class Terminal: ClassController() {
     private val tConfig = TerminalConfig()
     private val tBuilder = TerminalBuilder(tConfig)
     private val tabPane = TabPane()
-    private val placeholder = Placeholder(FontAwesomeIcon.TERMINAL, "New terminal (Ctrl+T)", { openNewTab() })
+    private val placeholder = Placeholder(FontAwesomeIcon.TERMINAL, "New terminal (${keys("CTRL+T")})", { openNewTab() })
```

Comment:
It is, it is only widget-scope shortcut so it is fine. It could optionally be refactored out to Action object that could be in the widget's settings but considering CTRL+T being common, I doubt there is a need. It should however be changed/reworded to support Mac.